src/Http.ts

```diff
@@ -18,7 +18,7 @@ export default class Http {
     this.host = process.env.SPREE_HOST || 'http://localhost:3000/'
```

Comment:
@tniezg if `process.env.SPREE_HOST` is not followed by a `/` then the absolute `storefrontPath` will not work. Thus, I would change the fallback to `http://localhost:3000` and append a `/` to the storefrontPath.

Reply:
@Shigawire if you could change it would be awesome - thank you!

Reply:
@Shigawire It works in both cases for me. What host did you try exactly? Was it `http://localhost:3000`?

Reply:
To be honest, I didn't try. But logically, using a host without trailing slash as `baseUrl` and then appending the `storefrontPath` without a leading `/` should be error-prone, not?

Reply:
Axios handles the case with missing `/`, so we can leave it as is.

Reply:
Fair enough then.